src/components/PreviewCompatibleImage.js

```diff
@@ -3,15 +3,22 @@ import PropTypes from "prop-types";
 import { GatsbyImage } from "gatsby-plugin-image";
 
 const PreviewCompatibleImage = ({ imageInfo }) => {
-  const { alt = "", image, wrapperStyle, url } = imageInfo;
-
+  const {
+    alt = "",
```

Comment:
I'd ask that we make this as close to required as possible and use the file name when it's missing